cmd/compute-domain-daemon/main.go

```diff
@@ -175,10 +182,22 @@ func run(ctx context.Context, cancel context.CancelFunc, flags *Flags) error {
 		computeDomainNamespace: flags.computeDomainNamespace,
 		nodeName:               flags.nodeName,
 		podIP:                  flags.podIP,
+		maxNodesPerIMEXDomain:  flags.maxNodesPerIMEXDomain,
 	}
 	klog.Infof("config: %v", config)
 
-	// Prepare IMEX daemon process manager (not invoking the process yet).
+	var dnsNameManager *DNSNameManager
+	if featuregates.Enabled(featuregates.IMEXDaemonsWithDNSNames) {
+		// Prepare DNS name manager
+		dnsNameManager = NewDNSNameManager(flags.cliqueID, flags.maxNodesPerIMEXDomain, nodesConfigPath)
```

Comment:
Let's add a log message here, explicitly acking that we take this code path and potentially emit some config detail.